sthree/registry.go

```diff
@@ -41,10 +41,12 @@ func newBucketRegistry() *bucketRegistry {
 
 func (r *bucketRegistry) init() {
 	auth, err := aws.EnvAuth()
-	grip.CatchDebug(err)
 	if err != nil {
-		auth, err = aws.SharedAuth()
-		grip.CatchDebug(err)
+		var err2 error
+		auth, err2 = aws.SharedAuth()
+		if err2 != nil {
+			grip.Debug("current does not provide aws credentials")
```

Comment:
Shouldn't this wrap?

Reply:
I think it's not clear which error we should wrap.

also this function is called in an init function, so I think less logging is probably better.